components/tidb_query/src/storage/range.rs

```diff
@@ -58,6 +58,17 @@ impl std::fmt::Debug for IntervalRange {
     }
 }
 
+impl Into<KeyRange> for IntervalRange {
```

Comment:
Try to implement `From<IntervalRange>` instead? It will implement `Into<KeyRange>` automatically.

Reply:
In our history code: https://github.com/tikv/tikv/blob/b7ddddba68b344127a0908034522c3d0306b3240/src/coprocessor/dag/mod.rs#L144

Here we just use `IntervalRange` to change our `KeyRange`. But now `KeyRange` may not involved `reverse` or not, so I don't know if it's proper to implement it.